pkg/version/version.go

```diff
@@ -23,3 +32,92 @@ func MaybePrintVersionAndExit(printVersion bool) {
 	}
 	log.Infof("running conduit version %s", Version)
 }
+
+type versionStatusChecker struct {
+	version string
+	client  pb.ApiClient
+}
+
+func (v versionStatusChecker) SelfCheck() (checks []*healthcheckPb.CheckResult) {
```

Comment:
We tend not to use named returns in the Conduit codebase. Would you mind making this a regular ("unamed"?
 how do you call those :P ? )return value?